scripts/jenkins/Jenkinsfile.release

```diff
@@ -10,166 +10,521 @@ properties([
       artifactDaysToKeepStr: '31',
       artifactNumToKeepStr: '10'
     )
-  )
+  ),
+  parameters([
+    booleanParam(
+      defaultValue: true,
+      description: 'If activated, changes to repositories are pushed. Can be combined with git_bundle.',
+      name: 'git_publish'
+    ),
+    booleanParam(
+      defaultValue: false,
+      description: 'If activated, changes to the ftp and doc repositories are bundled and archived in the publish stages. Can be combined with git_publish.',
```

Comment:
Do we need (two) parameters here?

Reply:
The indented purpose of these parameters was to make further changes to the git repo before changes are pushed. At least the second parameter needs to be definitely removed, since we always want an updated changelog after a package release. The more I think about it, the more unnecessary both parameter seem, so I will remove both of them.